src/RefractSourceMap.cc

```diff
@@ -0,0 +1,11 @@
+#include "RefractSourceMap.h"
+
+refract::IElement* drafter::BytesRangeToRefract(const mdp::BytesRange& bytesRange)
```

Comment:
Hmm so something is really wrong because 
```
    struct SourceMapBase {
        mdp::BytesRangeSet sourceMap;
    };
```

- there is wrong defined `SourceMapBase` or 
- conversion from `ByteRange` to `CharacterRange` is missing everywhere in drafter